WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/WalletBalanceTileViewModel.cs

```diff
@@ -7,12 +7,12 @@ namespace WalletWasabi.Fluent.ViewModels.Wallets.Home.Tiles;
 
 public class WalletBalanceTileViewModel : ActivatableViewModel
 {
-	public WalletBalanceTileViewModel(WalletViewModel walletVm)
+	public WalletBalanceTileViewModel(WalletViewModel walletVm, bool showOnlyAvailable = false)
 	{
 		var wallet = walletVm.Wallet;
 
-		var balance = walletVm.UiTriggers.BalanceUpdateTrigger
-			.Select(_ => wallet.Coins.TotalAmount());
+		var balance = showOnlyAvailable ? walletVm.UiTriggers.BalanceUpdateTrigger.Merge(walletVm.UiTriggers.WalletCoinsCoinjoinTrigger).Select(_ => wallet.Coins.Available().TotalAmount())
```

Comment:
Stick to the traditional format.

```suggestion
		var balance = showOnlyAvailable
			? walletVm.UiTriggers.BalanceUpdateTrigger.Merge(walletVm.UiTriggers.WalletCoinsCoinjoinTrigger).Select(_ => wallet.Coins.Available().TotalAmount())
			: walletVm.UiTriggers.BalanceUpdateTrigger.Select(_ => wallet.Coins.TotalAmount());
```